### PR TITLE
List Types of Loans

### PR DESCRIPTION
**Pull request by Jeya2621**
##### Types of Loans #####

### CODE REVIEW

File: src/main/java/in/jeya/service/DisplayLoans.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package in.jeya.service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DisplayLoans {
+
+	private static List<Loans> listOfLoans = new ArrayList<>();
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/jeya/service/DisplayLoans.java]: Add final variable and add private constructor

File: src/main/java/in/jeya/service/Loans.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package in.jeya.service;
+
+
+public class Loans {
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/jeya/service/Loans.java]: Rename to "Loan"

File: src/main/webapp/index.jsp [hunk 3]
```diff
@@ -1,7 +1,6 @@
 <%@page import="java.util.Date"%>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/index.jsp]: Remove unused imports.

File: src/test/java/in/jeya/service/DisplayTest.java [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+	}
+	
+	/*
+	 * For InValid Loan Types Display(using size)
+	 */
+
+	
+	//@Test
+	//public void test1Display() {
+	//	List<String> typesOfLoans = DisplayLoans.getLoanTypes();
```
Comment (csys-fresher-batch-2021) [inline src/test/java/in/jeya/service/DisplayTest.java]: Remove unused code.

**Review by ghost — CHANGES_REQUESTED** [review]
Review comments added.

File: src/main/java/in/jeya/service/Loan.java [hunk 5]
```diff
@@ -0,0 +1,24 @@
+package in.jeya.service;
+
+
+public class Loan {
+
+	private String loans;
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/jeya/service/Loan.java]: String name

File: src/main/webapp/Home.jsp [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+<head>
+<meta charset="ISO-8859-1">
+<title>Insert title here</title>
+</head>
+<body>
+	<div align = "center">
+	<strong><h2>!...Welcome...! </h2></strong>
+    <h2><strong><em>!!!.....ABC Financial Consultant.....!!!</em></strong></h2>
+    <h4><strong>No.17, 1st Street, Main Road, Madurai - 600004</strong></h4>
+    <h4><strong>Contact No. : 9876543210</strong></h4>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/Home.jsp]: Remove personal information.

File: src/main/webapp/Login.jsp [hunk 7]
```diff
@@ -0,0 +1,20 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+	pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html>
+
+<html lang = "en">
+<head>
+<meta charset="ISO-8859-1">
+<title>Insert title here</title>
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/Login.jsp]: Title?

File: src/main/webapp/Register.jsp [hunk 8]
```diff
@@ -0,0 +1,14 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/Register.jsp]: Delete this register.jsp

File: src/main/java/in/jeya/service/Loan.java (hunk 5 above)
Comment (csys-fresher-batch-2021) [reply src/main/java/in/jeya/service/Loan.java]: String type.

**Review by ghost — CHANGES_REQUESTED** [review]
Review comments added.

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**